internal/resto/structs.go

```diff
@@ -0,0 +1,36 @@
+package resto
+
+// Details represent API response regarding job
+type Details struct {
```

Comment:
maybe it needs format here :)

Reply:
You are right! I forgot to run go fmt. Now everything should be ok.